### ISSUE TITLE
U2U1206 when using overload with index

### ISSUE THREAD

**Issue by jnyrup** [u2uconsult/codeanalyzers#15]
This snippet triggers U2U1206 telling me to use an overload of `Any` with the same predicate instead

```csharp
Enumerable.Empty<int>()
	.Where((source, index) => true)
	.Any();
```

In `System.Linq` there is no overload of `Any` that takes a index parameter.

Am I missing something, or is this a false positive?

**Comment by KrisVandermotten** [comment]
This is a false positive indeed.

Thanks for reporting the bug, it will be fixed in the next release later this week.

**Comment by KrisVandermotten** [comment]
A fix is now available for download, in version 2.13.2.
